community.md

```diff
@@ -15,6 +15,7 @@ Below is a list of projects and resources from the Blazor community (links in ea
 * [BlazorNodaTimeDateTimePicker](https://github.com/nheath99/BlazorNodaTimeDateTimePicker) &ndash; A date picker using NodaTime.
 * [Blazorous](https://github.com/chanan/Blazorous) &ndash; Maintainable CSS with Blazor.
 * [BlazorSignalR](https://github.com/csnewman/BlazorSignalR) &ndash; ASP.NET Core SignalR Core .NET client support for Blazor.
+* [BlazorSplit](https://github.com/BlazorComponents/BlazorSplit) ([Demo](https://blazorcomponents.github.io/BlazorSplit/)) ([nuget.org](https://www.nuget.org/packages/BlazorSplit)) &ndash; Resizeable split views.
```

Comment:
We don't typically link to the NuGet package here. We've been linking primarily to the GitHub repo, and then optionally to a live demo site.

Reply:
I have removed nuget packages!